File: s3/cycle1/java/ex01.java
import java.util.*;

public class ex01{
	public static void main(String[] args){
		int i;
		Scanner input = new Scanner(System.in);
		System.out.print("Enter the number: ");
		int n = input.nextInt();
		for( i = 2 ; i <= n/2 ; i++)
		{
			if(n%i == 0){
				System.out.print("The number is not prime.\n");
				break;
			}
		}
		if(i >= n/2)
			System.out.print("The number is prime.\n");
	}
}

File: s3/cycle1/ex01.c
#include <stdio.h>
#include <math.h>

int main(void)
{
	int n,i;
	printf("Enter the number: ");
	scanf("%d" , &n);
	for(i = 2 ; i< sqrt(n) ; i++)
	{
		if(n%i == 0)
		{
			printf("\nGiven number is not a prime number\n");
			break;
		}
	}
	if(i >= sqrt(n))
		printf("\nThe given number is a prime number\n");
}